packages/edit-post/src/components/layout/index.js

```diff
@@ -232,22 +228,15 @@ function MetaBoxesMain( { isLegacy } ) {
 
 	const contents = (
 		<div
-			className={ clsx(
-				// The class name 'edit-post-layout__metaboxes' is retained because some plugins use it.
-				'edit-post-layout__metaboxes',
-				! isLegacy && 'edit-post-meta-boxes-main__liner'
-			) }
-			hidden={ ! isLegacy && isShort && ! isOpen }
+			// The class name 'edit-post-layout__metaboxes' is retained because some plugins use it.
+			className="edit-post-layout__metaboxes edit-post-meta-boxes-main__liner"
```

Comment:
Well, I think since we have to keep `edit-post-layout__metaboxes` for the sake of plugins we could utilize only that one. Is that what you mean? I suppose that’d be fine.

Reply:
That's right. I thought it would be nice to remove the `.edit-post-meta-boxes-main__liner` selector and put these styles into the `.edit-post-layout__metaboxes` selector:

https://github.com/WordPress/gutenberg/blob/d00892256f310540bb806e8b507c59332f99c6b9/packages/edit-post/src/components/layout/style.scss#L98-L107